tests/test_edit_log_path.py

```diff
@@ -0,0 +1,42 @@
+import json
+import re
+import sys
+from pathlib import Path
+from subprocess import run
+
+__author__ = "ZeroRin"
+log = Path.cwd()/'logs/log.txt'
+modified_log =  Path.cwd()/'logs/log.log'
+
+def test_edit_log_path():
+    config_file = Path("../logger_tt/log_config.json")
+    config = json.loads(config_file.read_text())
+    config['handlers']['urllib_file_handler'] = config['handlers']['error_file_handler']
```

Comment:
Please add a comment line so that we would know immediately what this line does 6 months later.